scripts/addFromConfig.py

```diff
@@ -279,17 +273,18 @@ def addStuff(cfg, options):
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-m", "--mission", type=str,
-                        help="mission to connect to", default='')
-    parser.add_argument("-v", "--verify", action='store_true',
-                        help="Don't do anything other than verify the config file", default=False)
-    parser.add_argument('config_file', action='store', type=str,
-                        help='Configuration file to read from')
+    usage = "usage: %prog [options] -m mission_db filename"
+    parser = OptionParser(usage=usage)
+    parser.add_option("-m", "--mission", dest="mission", type="string",
+                      help="mission to connect to", default='')
+    parser.add_option("-v", "--verify", dest="verify", action='store_true',
+                      help="Don't do anything other than verify the config file", default=False)
 
-    options = parser.parse_args()
+    (options, args) = parser.parse_args()
+    if len(args) != 1:
+        parser.error("incorrect number of arguments")
 
-    filename = os.path.expanduser(options.config_file)
+    filename = os.path.expanduser(args[0])
```

Comment:
Why port this back to optparse, which is deprecated, instead of argparse, which isn't? (argparse isn't available on 2.6, but that's way back.)

Reply:
Not sure what happened here.  I did struggle with the arguments in some functions.  It's seem that I could not get the config file.

Reply:
Okay, I'll revert this bit then.